### ISSUE TITLE
Working on linux - not working on Windows server 2012

### ISSUE THREAD

**Issue by SOPhorrific** [marcelo-mason/PreciousStones#1078]
I use a custom jar (same one for both server)
same exact config files on both

https://hastebin.com/pexutifoso.md

and this makes AMP never resolve the start of the server

as it's stuck on precious stones

If I disable it my server works

any idea?

**Comment by SOPhorrific** [comment]
Using SQLite.db Not WAMP one on my dedicated server running WS2012

**Comment by marcelo-mason** [comment]
Turn on debug mode in the PS settings for console output of whats going on there.

**Comment by SOPhorrific** [comment]
it is already enabled

**Comment by marcelo-mason** [comment]
Weird,  the next thing after that last message is an asynchronous call, that apparently never runs since the next log output never shows up.

https://github.com/marcelo-mason/PreciousStones/blob/c9ea3d3528ef4d28b0d12e347d35e6bc2d455cac/src/main/java/net/sacredlabyrinth/Phaed/PreciousStones/managers/StorageManager.java#L350

**Comment by marcelo-mason** [comment]
Could be interference with another plugin, or something in your environment messing up that async call.

**Comment by SOPhorrific** [comment]
So any idea?
I mean on linux it works, and I'm using AMP mcmyadmin3 and windows server 2012 same everything

the db have same weight
the .jar is the same version

I'm just so confused

I opened the .db to see if it had be corrupted during transfer and everything seems clean

**Comment by marcelo-mason** [comment]
No idea, could be anything ,no clue.  Try taking out all other plugins seeing if it works just with PS, then add them back bit by bit.

**Comment by SOPhorrific** [comment]
23:04:22 INFO]: Starting minecraft server version 1.11.2
[23:04:22 INFO]: Loading properties
[23:04:22 WARN]: resource-pack-hash is deprecated and found along side resource-pack-sha1. resource-pack-hash will be ignored.
[23:04:22 WARN]: You specified a resource pack without providing a sha1 hash. Pack will be updated on the client only if you change the name of the pack.
[23:04:22 INFO]: Default game type: SURVIVAL
[23:04:22 INFO]: This server is running Paper version git-Paper-1056 (MC: 1.11.2) (Implementing API version 1.11.2-R0.1-SNAPSHOT)
23:04:23[23:04:23 INFO]: Debug logging is disabled
[23:04:23 INFO]: Using -1 threads for Netty based IO
[23:04:23 INFO]: Server Ping Player Sample Count: 12
[23:04:23 INFO]: Spigot Timings: false - Verbose: true - Interval: 5m - Length: 60m
[23:04:23 INFO]: Generating keypair
[23:04:23 INFO]: Starting Minecraft server on 107.175.106.208:25565
[23:04:23 INFO]: Using default channel type
[23:04:23 INFO]: Set PluginClassLoader as parallel capable
[23:04:23 INFO]: [PreciousStones] Loading PreciousStones v10.0.5
[23:04:23 INFO]: Server permissions file permissions.yml is empty, ignoring it
[23:04:23 INFO]: **** Beginning UUID conversion, this may take A LONG time ****
[23:04:23 INFO]: Preparing level "custom"
23:04:24[23:04:24 INFO]: Preparing start region for level 0 (Seed: 27594263)
23:04:25[23:04:25 INFO]: Preparing spawn area: 22%
[23:04:25 INFO]: Preparing start region for level 1 (Seed: 3532872949660453992)
23:04:26[23:04:26 INFO]: Preparing start region for level 2 (Seed: 3532872949660453992)
[23:04:26 INFO]: [PreciousStones] Enabling PreciousStones v10.0.5
[23:04:26 INFO]: [PreciousStones] Version 10.0.5 loaded
[23:04:26 INFO]: [PreciousStones] SQLite Connection successful
[23:04:26 INFO]: finalizing queue
[23:04:26 INFO]: clearing fields from memory

**Comment by SOPhorrific** [comment]
Same error, no plugin except 10.0 5 (is the nathan wolf modified version for magic, and is working on linux)

**Comment by marcelo-mason** [comment]
Try upgrading PS, the one youre on is kinda old.

http://104.236.246.108:8080/job/PreciousStones/

**Comment by SOPhorrific** [comment]
I dont think you kept nathanwolf dependency for the recall spell

**Comment by marcelo-mason** [comment]
What you mean?  Where can I get that version, maybe I'll add those mods to the main one.

**Comment by SOPhorrific** [comment]
So even when I wipe all my plugins use your lastest one


23:10:08[23:10:08 INFO]: Preparing start region for level 2 (Seed: 3532872949660453992)
[23:10:08 INFO]: [PreciousStones] Enabling PreciousStones v10.5.3
[23:10:08 INFO]: [PreciousStones] Version 10.5.3 loaded
[23:10:08 INFO]: [PreciousStones] SQLite Connection successful
[23:10:08 INFO]: finalizing queue
[23:10:08 INFO]: clearing fields from memory




and my server freezes

**Comment by SOPhorrific** [comment]
and my server freezes

**Comment by marcelo-mason** [comment]
What happens when you take out the sqllite.db file?

**Comment by SOPhorrific** [comment]
https://hastebin.com/ponosibana.sql
Create a new db

still freezes

**Comment by marcelo-mason** [comment]
Man no clue, could be some weird thing with Paper.

**Comment by SOPhorrific** [comment]
do you have an update checker on precious stones?
if yes can I disable it?

I'm pretty sure I know why

**Comment by marcelo-mason** [comment]
I don't

On Jul 1, 2017 3:44 AM, "SOPhorrific" <notifications@github.com> wrote:

do you have an update checker on precious stones?
if yes can I disable it?

I'm pretty sure I know why

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/marcelo-mason/PreciousStones/issues/1078#issuecomment-312417283>,
or mute the thread

**Comment by frizzbee30** [comment]
We run it on windows serer 2012 for our test box, with an external MySQL db link with no issues.. (standard spigot)

**Comment by SOPhorrific** [comment]
Well mine is SQLite

**Comment by SOPhorrific** [comment]
menu
Status #paperX
[Source: https://github.com/PaperMC/Paper | Maven: https://repo.destroystokyo.com/repository/maven-public/ | Download: https://ci.destroystokyo.com/job/PaperSpigot/ | High performance Spigot fork that aims to fix gameplay and mechanic inconsistencies]
[18:35] <SOPhorrific> hey @Electronicat...... when you did run your instance of WS 2012 w/ AMP did you use precioustones compile format in MYSql or SQLite
[18:35] <SOPhorrific> I do not use WAMP
[18:35] <SOPhorrific> that might be why
[18:35] <Mana> [%electroniccat] sqlite
[18:35] <SOPhorrific> ok thx

**Comment by SOPhorrific** [comment]
should work both instance, and plus I know that WAMP is not optimum, better run LAMP on the darkweb using an old cellphone running a dependency with linux on it

so go figure :)

thanks for the advice tho :)

**Comment by SOPhorrific** [comment]
By the way LAMP is the less requirements on earth
wamp is the worse 

and tbh worse of what?

lol.. dont do that stick with WAMP ;)


Any other toughts people?

**Comment by SOPhorrific** [comment]
gonna put ubuntu lets hope it wil lsolve it

**Comment by MDelgadoCCM** [comment]
I doubt your problem is the stack.  Plenty of people run PS in Windows, I do myself.

**Comment by marcelo-mason** [comment]
I doubt your problem is the stack. Plenty of people run PS in Windows, I do myself.  I have a feeling something in Paper is messing it up.  Try Spigot or Bukkit, if that works then you know its Paper, then you can start messing with the settings to see what could be it.

**Comment by dsevillamartin** [comment]
Hi. I've been working with SOPhorrific to set up the server.

We've changed to Ubuntu 17.10, however, the same exact issues have occurred. I've tried to download Spigot, but it just gets stuck after downloading something.
I believe the hosting service may be blocking some endpoints, because some websites (like Java.com) would never load.

Paper is not what's messing it up, because I uploaded to the server the exact same files I use in Mac for a testing server, and PreciousStones did not work either. We are also having an issue with EpicWorldGenerator, so we have no idea what's causing the problems.

I tried to use PreciousStones with MySQL, but it got stuck right after it added the new indexes to the database.

Any information would be useful.
Thank you for your time.
